File: R23/R23.py
def calculator_int(saving,interest,time):
    total = saving * (1+interest/100)**time
    return round(total,2)



def compound_interest():
    saving = 100
    interest = int(input('¿Cual será el interes compuesto?'))

    interest_time = {
        '6 months': calculator_int(saving,interest,0.5),
        '1 year': calculator_int(saving,interest,1),
        '2 year': calculator_int(saving,interest,2),
    }

    for i,j in interest_time.items():
        print(f'{i} => {j}')

compound_interest()

File: R23/R23.js

const calculator = (saving,interest,time) =>{
    let total = saving * (1+interest/100)**time
    return total.toFixed(2)
}

const compound_interest = () =>{
    const saving = 100;
    const interest = int(prompt('¿Cuanto será el interes?'));
    let total_interest = {
        '6 meses' : calculator(saving,interest,0.5),
        '1 año' : calculator(saving,interest,1),
        '2 años' : calculator(saving,interest,2)
    }
    console.log(total_interest)

}

compound_interest()